examples/saas-dynamic-dashboards/frontend/components/data-chat-grid.tsx

```
@@ -0,0 +1,152 @@
+
+    const handleRowSelect = (index: number, checked: boolean) => {
+        if (checked) {
+            setSelectedRows([...selectedRows, index]);
+        } else {
+            setSelectedRows(selectedRows.filter(i => i !== index));
+        }
+    };
+
+    const handleSelectedAction = () => {
+        // Handle the action for selected rows
+        debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)